portworx/README.md

```diff
@@ -0,0 +1,93 @@
+#### Preparing your EKS Cluster
+
+Before we can create an IAMServiceAccount for Portworx so we can send metering data to AWS.
+we need to enable the IAM OIDC Provider for your EKS cluster.
+Make sure to replace `<clustername>` with your EKS cluster and change the `region` if you are not running in us-east-1
+```
+eksctl utils associate-iam-oidc-provider --region=us-east-1 --cluster=<clustername> --approve
+```
+
+Second we have to create the IAMServiceAccount with the appropriate permissions.
+Make sure to change the namespace if you are not deploying in `kube-system` and make sure to replace `<clustername>` with your EKS cluster
+```
+eksctl create iamserviceaccount --name portworx-aws --namespace kube-system --cluster <clustername> --attach-policy-arn arn:aws:iam::aws:policy/AWSMarketplaceMeteringFullAccess \
+--attach-policy-arn arn:aws:iam::aws:policy/AWSMarketplaceMeteringRegisterUsage --approve --override-existing-serviceaccounts
+```
+
+This will create an `IAMServiceAccount` on amazon `https://console.aws.amazon.com/iam/home?#/roles` and
+will create a `ServiceAcccount` in the requested namespace, which we will pass to our helmchart.
+
+
+#### Installation
+To add the Portworx AWS Helm repository run the following command:
+```
+helm repo add portworx https://raw.githubusercontent.com/portworx/aws-helm/master/stable
```

Comment:
Can you ask Shailvi to create an alias as we have for our public helm chart, something like https://aws-chart.portworx.io

Reply:
Sure I can do that. Any reason the existing PX is using this?

Reply:
Have requested Shaivi to create an alias, for now pushing the PR as there are changes for product ID to be done once we have the actual product in place during which will address this also.